JSDemos/Demos/Slider/Overview/Angular/description.md

```diff
@@ -0,0 +1,33 @@
+The Slider component allows users to set a numeric value on a continuous range of possible values. This demo shows how you can create and configure a Slider.
+
+## Create a Slider
+
+To create a Slider, declare it in markup and use the [min](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#min) and [max](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#max) properties to limit the range of accepted values.
+
+You can also specify the [value](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#value) property to change the initial value.
+
+You can see the resulting Slider in the **Default mode** section of this demo.
+
+## Customize Slider Appearance
+
+The Slider can display labels for the [min](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#min) and [max](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#max) values. To configure the labels, use the [label](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/label/) object. In this object, enable the [visible](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/label/#visible) property to display the labels. You can also specify the [position](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/label/#position) and [format](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/label/#format) properties.
```

Comment:
I first noticed two similar phrases in consecutive sentences: "To configure the labels", "to display the labels". If I want to configure them, I'll probably display them. So that's one reason I wouldn't want to say the second one. In addition to that, the name `visible` is self-explanatory. So I want to cut some text. Maybe use this:

```suggestion
The Slider can display labels for the [min](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#min) and [max](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/#max) values. To configure the labels, use the [label](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/label/) object. In this object, set the [visible](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/label/#visible), [position](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/label/#position), and [format](/Documentation/ApiReference/UI_Components/dxSlider/Configuration/label/#format) properties.
```